creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -159,6 +163,50 @@ const syncRouteController = async (req, res) => {
   return successResponse()
 }
 
+/**
+ * Adds a job to manualSyncQueue to issue a sync to secondary with syncMode MergePrimaryAndSecondary
+ * @notice This will only work if called on a primary for a user
+ */
+const mergePrimaryAndSecondaryController = async (req, res) => {
```

Comment:
this doesn't need auth right? probably fine since the function itself has auth

Reply:
ya i was debating putting an `ensurePrimaryMiddleware` on this - it is prob fine but now thinking prob just add for safety right - wdyt?

Reply:
are we also checking anywhere to make sure it's signed by a valid SP so that not just anyone can access the route, or is it better to leave that out for easier testing?

Reply:
per slack discussion, made linear note to circle back to this